qml/components/ChumDetailItem.qml

```diff
@@ -7,10 +7,11 @@ Label {
 
   anchors {
     left: parent.left
-    leftMargin: Theme.horizontalPageMargin
     right: parent.right
     rightMargin: Theme.horizontalPageMargin
```

Comment:
It is good idea to drop leftMargin. But then let's remove the rightMargin as well for consistency. For ChumDetailItem used outside of columns that have margins already, we can specify the both margins.